test/js/actions.unit.spec.js

```diff
@@ -129,31 +129,6 @@ describe('Schemaform actions:', () => {
 
       return promise;
     });
-    it('should submit with auth header', () => {
```

Comment:
This test is no longer needed. _BUT_ it would be ideal to add a test that either:

- checks that Cookies are added to the request, or...
- checks that we configured the XHR with `withCredentials`

Reply:
Removing it is fine since it really shouldn't have been there anyway, I did it in a [later commit](https://github.com/usds/us-forms-system/commit/37c569781d188be421eed28e9b5987aeea3e6370#diff-731cfe914af9eff1a334a7c9ab16bcd6L132) for 2.0.  There's not an easy way to verify that cookies were sent to the server so let's punt on that.